libbpf-tools/kern_map_helpers.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2020 Anton Protopopov
+#ifndef __KERN_MAP_HELPERS_H
+#define __KERN_MAP_HELPERS_H
```

Comment:
I suggest following .bpf.h convention for BPF-only headers, we already have bits.bpf.h, so this one could be maps.bpf.h, WDYT?

Reply:
Sure, this looks better